tools/deployments/validate-pr-description.ts

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable turbo/no-undeclared-env-vars */
+if (require.main === module) {
+	const errors = validateDescription(
+		process.env.TITLE as string,
+		process.env.BODY as string,
+		process.env.LABELS as string
+	);
+	if (errors.length > 0) {
+		console.error("Validation errors in PR description:");
+		for (const error of errors) {
+			console.error("- ", error);
+		}
+		process.exit(1);
+	}
+}
+
+export function validateDescription(
+	title: string,
+	body: string,
+	labels: string
+) {
+	const errors: string[] = [];
+
+	console.log("PR:", title);
+	const parsedLabels = JSON.parse(labels);
+
+	if (parsedLabels.includes("skip-pr-validation")) {
```

Comment:
No one should ever really use this, but in the event we _do_ need to skip validation I added this so that it doesn't require an admin override